tools/run_test_suite.bash

```diff
@@ -5,10 +5,8 @@ set -ex
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"  # gets the directory of this script
 
 # Skip flaky tests. Nav2 system tests will be run later.
-colcon test --packages-skip nav2_system_tests nav2_dynamic_params nav2_motion_primitives \
-                            nav2_bt_navigator nav2_costmap_2d nav2_lifecycle_manager \
-                            nav2_mission_executor nav2_tasks nav2_robot
-                            
+colcon test --packages-skip nav2_system_tests nav2_dynamic_params nav2_motion_primitives
+
```

Comment:
It's still failing some linter tests that you reenabled.

Reply:
This could be from before #839 was merged. It passes on mine after rebase.

Reply:
Passing now